### PR TITLE
update screenshots on API Security

### PR DESCRIPTION
**Pull request by dushaniw**

### CODE REVIEW

File: en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Each tenant can define their name for the authorization header by following the
 
 2.  Click **Main**, then click **Browse** which is under **Resources**.
 
-     [![]({{base_path}}/assets/attachments/103334899/103334897.png)]({{base_path}}/assets/attachments/103334899/103334897.png)
+     [![]({{base_path}}/assets/img/learn/registry-browse.png)]({{base_path}}/assets/img/learn/registry-browse.png)
```
Comment (Mariangela) [inline en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md:94]: Please add ALT tags inside [ ]

Comment (Mariangela) [inline en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md:94]:
```suggestion
     [![Browse registry]({{base_path}}/assets/img/learn/registry-browse.png)]({{base_path}}/assets/img/learn/registry-browse.png)
```

**Review by Mariangela — APPROVED** [review]

File: en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md (hunk 1 above)
Comment (dushaniw) [reply en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md]: fixed https://github.com/wso2/docs-apim/pull/3716/commits/58d252dcbb7d3f9308de62fbaa707a17a31ad144

Comment (dushaniw) [reply en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md]: resolved https://github.com/wso2/docs-apim/pull/3716/commits/58d252dcbb7d3f9308de62fbaa707a17a31ad144

File: en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md [hunk 2]
```diff
@@ -256,4 +257,4 @@ curl -H "SERVER_AUTH: Bearer 3c536e3f-397c-3df9-a89c-9c40efedfa9e" -H "accept: a
 
 Note that the customized authorization header appears in the Developer Portal UI as well.
 
-[![]({{base_path}}/assets/img/learn//auth_header.png)]({{base_path}}/assets/img/learn//auth_header.png)
+[![Customized Auth Header]({{base_path}}/assets/img/learn//auth_header.png)]({{base_path}}/assets/img/learn//auth_header.png)
```
Comment (Mariangela) [inline en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md:260]:
```suggestion
[![Customized Auth Header]({{base_path}}/assets/img/learn/auth_header.png)]({{base_path}}/assets/img/learn/auth_header.png)
```

File: en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md [hunk 3]
```diff
@@ -211,11 +211,12 @@ Follow the instructions below to add a customized authorization header for an AP
 
 1.  [Sign in to the Publisher]({{base_path}}/install-and-setup/install/running-the-product/#accessing-the-api-publisher).
 `https://<server-host>:9443/publisher         `
-2.  When [creating]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) or updating an API, define the customized authorization header in the **Runtime Configurations** tab.
 
-    [![]({{base_path}}/assets/img/learn/custom-auth-header-publisher.png)]({{base_path}}/assets/img/learn//custom-auth-header-publisher.png)
+2.  When [creating]({{base_path}}/design/create-api/create-rest-api/create-a-rest-api/) or updating an API, define the customized authorization header in the **Runtime** tab under **Develop -> API Configurations**.
 
-3.  Save and Publish the API.
+    [![Custom Auth Header]({{base_path}}/assets/img/learn/custom-auth-header-publisher.png)]({{base_path}}/assets/img/learn//custom-auth-header-publisher.png)
```
Comment (Mariangela) [inline en/docs/design/api-security/api-authentication/secure-apis-using-oauth2-tokens.md:217]:
```suggestion
    [![Custom Auth Header]({{base_path}}/assets/img/learn/custom-auth-header-publisher.png)]({{base_path}}/assets/img/learn/custom-auth-header-publisher.png)
```

File: en/docs/design/api-security/api-authentication/secure-apis-using-basic-authentication.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ Basic Authentication is an API level configuration. Please sign in to the API Pu
 Basic Authentication. Under the Application Level Security section in the Runtime Configuration of the API Details page allow users to configure
 the Basic Authentication as follow.
 
-[ ![]({{base_path}}/assets/img/learn//basic_authentication.png) ]({{base_path}}/assets/img/learn//basic_authentication.png)
+[![Basic Authentication]({{base_path}}/assets/img/learn//basic_authentication.png)]({{base_path}}/assets/img/learn//basic_authentication.png)
```
Comment (Mariangela) [inline en/docs/design/api-security/api-authentication/secure-apis-using-basic-authentication.md:14]:
```suggestion
[![Basic Authentication]({{base_path}}/assets/img/learn/basic_authentication.png)]({{base_path}}/assets/img/learn/basic_authentication.png)
```

**Review by Mariangela — APPROVED** [review]

**Pull request merged**